build/ci.go

```diff
@@ -282,17 +282,24 @@ func doInstall(cmdline []string) {
 
 func buildFlags(env build.Environment) (flags []string) {
 	var ld []string
+
```

Comment:
This is quite obvious, but we will have to take care about this during every rebase with upstream.